rcp/org.testeditor.rcp4.views/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,104 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Views
+Bundle-SymbolicName: org.testeditor.rcp4.views;singleton:=true
+Bundle-Version: 1.0.0.qualifier
+Bundle-Vendor: Signal Iduna Corporation
+Require-Bundle: com.google.guava,
+ org.eclipse.xtext.xbase.lib,
+ org.eclipse.xtend.lib,
+ org.eclipse.xtend.lib.macro,
+ org.eclipse.core.runtime,
+ org.eclipse.swt,
+ org.eclipse.e4.ui.model.workbench,
+ org.eclipse.ui.editors,
+ org.eclipse.xtext.xbase.ui,
+ org.eclipse.jface
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Import-Package: com.google.inject,
+ org.eclipse.e4.core.di,
+ org.eclipse.e4.core.di.annotations,
+ org.eclipse.e4.core.di.extensions,
+ org.eclipse.e4.core.di.internal.extensions,
+ org.eclipse.e4.core.di.suppliers,
+ org.eclipse.e4.core.services.events,
+ org.eclipse.e4.ui.di,
+ org.eclipse.e4.ui.model.application.ui.advanced,
+ org.eclipse.e4.ui.model.application.ui.basic,
+ org.eclipse.e4.ui.workbench,
+ org.eclipse.e4.ui.workbench.modeling,
+ org.eclipse.emf.common.notify,
+ org.eclipse.emf.common.util,
+ org.eclipse.emf.ecore,
+ org.eclipse.emf.ecore.action,
+ org.eclipse.emf.ecore.change,
+ org.eclipse.emf.ecore.change.impl,
+ org.eclipse.emf.ecore.change.provider,
+ org.eclipse.emf.ecore.change.util,
+ org.eclipse.emf.ecore.impl,
+ org.eclipse.emf.ecore.plugin,
+ org.eclipse.emf.ecore.presentation,
+ org.eclipse.emf.ecore.provider,
+ org.eclipse.emf.ecore.resource,
+ org.eclipse.emf.ecore.resource.impl,
+ org.eclipse.emf.ecore.util,
+ org.eclipse.emf.ecore.xcore,
+ org.eclipse.emf.ecore.xcore.exporter,
+ org.eclipse.emf.ecore.xcore.exporter.ui,
+ org.eclipse.emf.ecore.xcore.formatting,
+ org.eclipse.emf.ecore.xcore.generator,

```

Comment:
result of experimenting with lots of different apis. will have to cleanup those ...